fli-iam/shanoir-ng
# #2364-github-packages

- Update docker-compose file to use generated github packages.

How to test:
- Restart shanoir from scratch and check that every container works.
- 
Closes #2364
